### PR TITLE
Update version: bitvise.SSHClient 8.49

### PR DESCRIPTION
**Pull request by quhxl**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/23822)

### CODE REVIEW

**Review by ItzLevvie — DISMISSED** [review]

**Review by vedantmgoyal9 — APPROVED** [review]

**Comment by jamespik** [comment]
Why is the alternate download link being used rather than the primary as indicated on the bitvise website?

File: manifests/b/bitvise/SSHClient/8.49/bitvise.SSHClient.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+- ssh
+FileExtensions:
+- tlp
+- bscp
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: exe
+  Scope: machine
+  InstallerUrl: https://s3.amazonaws.com/dl.bitvise.com/BvSshClient-Inst.exe
```
Comment (vedantmgoyal2009) [inline manifests/b/bitvise/SSHClient/8.49/bitvise.SSHClient.installer.yaml:19]:
```suggestion
  InstallerUrl: https://dl.bitvise.com/BvSshClient-Inst.exe
```

**Review by vedantmgoyal9 — CHANGES_REQUESTED** [review]

File: manifests/b/bitvise/SSHClient/8.49/bitvise.SSHClient.installer.yaml (hunk 1 above)
Comment (quhxl) [reply manifests/b/bitvise/SSHClient/8.49/bitvise.SSHClient.installer.yaml:19]: Why should I change that? The distribution via AWS is surely more stable.

Comment (denelon) [reply manifests/b/bitvise/SSHClient/8.49/bitvise.SSHClient.installer.yaml:19]: We have policies regarding the domain an installer comes from. Automated checks will trigger. It's also possible the URL suggested by @vedantmgoyal2009 is a redirect, so the binary location may move, and the s3 bucked might be dropped while the dl.bitvise.com URL stays consistent and redirects to the new location.

**Review by vedantmgoyal9 — COMMENTED** [review]
Just because of the alternate download link, it was getting a "Validation-Domain" error.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**